### ISSUE TITLE
conditional-comment

### ISSUE THREAD

**Issue by AndreyGladkov** [bem/bem-forum-content-ru#429]
Можно ли  добавить условие в bemjson что бы блок написаный мной показывался только в  IE > 9,
по аналогии 
```
{
	elem : 'conditional-comment',
	condition : '< IE 9',
	content : [
		{ elem : 'js', url : '//yastatic.net/es5-shims/0.0.1/es5-shims.min.js' },
		{ elem : 'js', url : '/js/modernizr-html5shiv-respond.min.js' }
	]
},
```

**Comment by Guria** [comment]
Прятать его с помощью css везде кроме IE > 9?

**Comment by AndreyGladkov** [comment]
Нет нет))  - это canvas
Я могу и в js проверять по юзер агенту, просто думал может есть встроенный механизм

**Comment by Guria** [comment]
Т.е. вы хотите чтобы шаблон блока не разворачивался для браузеров  IE <=9 ?

**Comment by AndreyGladkov** [comment]
да, в моих снах бэм все может)) Вообщем я видимо переработался.

**Comment by Guria** [comment]
Кажется вы сами догадались, что обычно bemjson обрабатывается вне контекста браузера.
Ваша задача решается на уровне приложения, которое исключит не нужный блок перед обработкой.
